python/grass/script/tests/conftest.py

```diff
@@ -0,0 +1,20 @@
+"""Fixtures for grass.script"""
+
+import pytest
+
+
+@pytest.fixture
+def mock_no_session(monkeypatch):
+    """Set the environment variables as if there would be no background session.
+
+    Use with usefixtures (not as a paramter) to avoid warnings about an unused
+    parameter::
+
+        @pytest.mark.usefixtures("mock_no_session")
+        def test_session_handling():
+            pass
```

Comment:
My ReStructuredText is really rusty, 5-8 years, but wasn't code blocks defined with two semicolons on a line, with the correct indentation?

Something like this (don't commit as suggestion if it isn't right)
```suggestion
    Use with usefixtures (not as a paramter) to avoid warnings about an unused
    parameter

::

        @pytest.mark.usefixtures("mock_no_session")
        def test_session_handling():
            pass
```

Reply:
There is a shorthand syntax using that double colon instead of a single colon at the end of the paragraph.